style.css

```diff
@@ -26,4 +26,10 @@ Text Domain: startertheme
 .pie-footer ul, li {
   list-style: none;
   padding-left: 0;
+}
+
+
+/* CONTACT FORMS */
+.cf-holder {
+  width: 90%;
```

Comment:
Try using a bootstrap class for this too, but if those don't work out, you can stick with 90%.